python/pysrc/kaskada/_execution.py

```diff
@@ -38,6 +41,39 @@ class _ExecutionOptions:
     #: For snapshots, this determines when the snapshot is produced.
     final_at: Optional[int] = None
```

Comment:
Can you (or I can add later) specify the units for these options

Reply:
Let's do that later -- this just moved the code so we could use it for execute and rendering, don't think we should pull in extra changes.